src/components/Tile.js

```diff
@@ -0,0 +1,42 @@
+import { useGameContext } from '../context/GameContext.js';
+import './Tile.css';
+// import horse from '../../public/horse.png';
+// import snake from '../../public/snake.png';
+export default function Tile({ tileContent, index }) {
+  const { player, setPlayer, board, active, setBoard, setMessage } = useGameContext();
+
+  function handleClick() {
+    if (tileContent === 'Snake') {
+      return;
+    }
+    if (tileContent === 'Horse') {
```

Comment:
you could've just done ```if (tileContent) return``` as since there are only three options for tile content (null / empty string, 'Snake', or 'Horse'), and you want to exit if there is content (ie a non-empty string) in tileContent